src/sage/matroids/graphic_matroid.py

```diff
@@ -1357,26 +1397,29 @@ def graphic_extension(self, u, v=None, element=None):
 
         OUTPUT:
 
-        A GraphicMatroid with the specified element added. Note that if ``v`` is not
-        specifies or if ``v`` is ``u``, then the new element will be a loop. If the
-        new element's label is not specified, it will be generated automatically.
+        A GraphicMatroid with the specified element added. Note that if ``v``
+        is not specifies or if ``v`` is ``u``, then the new element will be a
+        loop. If the new element's label is not specified, it will be
+        generated automatically.
 
         EXAMPLES::
 
             sage: M = matroids.CompleteGraphic(4)
             sage: M1 = M.graphic_extension(0,1,'a'); M1
             Graphic matroid of rank 3 on 7 elements
             sage: list(M1.graph().edge_iterator())
-            [(0, 1, 'a'), (0, 1, 0), (0, 2, 1), (0, 3, 2), (1, 2, 3), (1, 3, 4), (2, 3, 5)]
+            [(0, 1, 'a'), (0, 1, 0), (0, 2, 1), (0, 3, 2), (1, 2, 3),
+            (1, 3, 4), (2, 3, 5)]
             sage: M2 = M1.graphic_extension(3); M2
             Graphic matroid of rank 3 on 8 elements
 
         ::
 
             sage: M = Matroid(range(10), graphs.PetersenGraph())
-            sage: sorted(M.graphic_extension(0, 'b', 'c').graph().vertex_iterator(), key=str)
+            sage: EX_G = M.graphic_extension(0, 'b', 'c').graph()
+            sage: sorted(EX_G.vertex_iterator(), key=str)
             [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 'b']
-            sage: M.graphic_extension('a', 'b', 'c').graph().vertices(sort=False)
+            sage: EX_G = M.graphic_extension('a', 'b', 'c').graph()
```

Comment:
```suggestion
            sage: M.graphic_extension('a', 'b', 'c').graph()
```
No need to suggest it is being stored.